Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -862,6 +863,18 @@ invalidate_stats()
   self_intersect = false;
 }
 
+Scene_polyhedron_item::Vertex_selection_map 
+Scene_polyhedron_item::vertex_selection_map()
+{
+  return get(boost::vertex_index,*d->poly);
+}
+
+Scene_polyhedron_item::Face_selection_map 
+Scene_polyhedron_item::face_selection_map()
+{
+  return get(boost::face_index,*d->poly);
```

Comment:
In the original code the selection status was stored in the index which is a writable property of the enriched polyhedron type. So I just sticked to the index as I was not sure if there are any side effects if we write in the selection property.